crates/oxc_linter/src/options.rs

```diff
@@ -71,46 +126,48 @@ impl LintOptions {
     pub fn derive_rules(&self) -> Vec<RuleEnum> {
         let mut rules: FxHashSet<RuleEnum> = FxHashSet::default();
 
-        for (allow_warn_deny, name_or_category) in &self.filter {
-            let maybe_category = RuleCategory::from(name_or_category.as_str());
-            match allow_warn_deny {
-                AllowWarnDeny::Deny => {
-                    match maybe_category {
-                        Some(category) => rules.extend(
-                            RULES.iter().filter(|rule| rule.category() == category).cloned(),
-                        ),
-                        None => {
-                            if name_or_category == "all" {
-                                rules.extend(RULES.iter().cloned());
-                            } else {
-                                rules.extend(
-                                    RULES
-                                        .iter()
-                                        .filter(|rule| rule.name() == name_or_category)
-                                        .cloned(),
-                                );
-                            }
-                        }
-                    };
-                }
-                AllowWarnDeny::Allow => {
-                    match maybe_category {
-                        Some(category) => rules.retain(|rule| rule.category() != category),
-                        None => {
-                            if name_or_category == "all" {
-                                rules.clear();
-                            } else {
-                                rules.retain(|rule| rule.name() != name_or_category);
-                            }
-                        }
-                    };
-                }
-            }
-        }
+        parse_eslint_config(self.config_path.clone().unwrap().clone()).unwrap()
+
+        // for (allow_warn_deny, name_or_category) in &self.filter {
```

Comment:
Do you have any plans for this commented function? Seems I need to continue work on here for adding support of `--jsx-a11y-plugin` and `--jest-plugin` 🤔

https://github.com/web-infra-dev/oxc/issues/1138

Reply:
yeah - i think we should override the parsed eslint config with those?

`filter` comes from the command line arguments right